### DISCUSSION TITLE
Slowlog detection?

### DISCUSSION THREAD

**Discussion by dmitryuk** [roadrunner-server/roadrunner#1847] [Q&A] [answered]
php-fpm has automatic slowlog detection with trace data (callstack).
Does RR has the same functionality?

**Comment by rustatian** [comment, marked as answer]
Hey @dmitryuk 👋 
Inside a `pool.supervisor` we have a `exec_ttl`, which is not actually the same feature, but helps you to automatically stop and restart slow workers. RR also supports OTEL to provide a call stack: [link](https://docs.roadrunner.dev/logging-and-observability/otel)
